packages/workflow/src/TypeValidation.ts

```diff
@@ -145,6 +145,17 @@ export const tryToParseObject = (value: unknown): object => {
 	}
 };
 
+export const getValueDescription = <T>(value: T, stringify = false): string => {
```

Comment:
`stringify` doesn't seem to be passed in anywhere so far. We should remove it, and also remove line 151, until they are actually needed.